community/qemu/build

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh -e
+
+patch -p0 < 001-map-sync.patch
+patch -p0 < 002-ppc.patch
+patch -p0 < 003-sigtrm-max.patch
+patch -p0 < 004-syscall.patch
+patch -p0 < 005-elfload.patch
+
+./configure \
+    --prefix=/usr \
+    --localstatedir=/var \
+    --sysconfdir=/etc \
+    --python=/usr/bin/python3 \
```

Comment:
Is `python` needed for build? If so, it should be a dependency.

Reply:
Added python as a dependency.